ocis/pkg/init/init.go

```diff
@@ -83,6 +83,10 @@ type FrontendService struct {
 	ServiceAccount ServiceAccount `yaml:"service_account"`
 }
 
+type OcmService struct {
```

Comment:
exported type `OcmService` should have a comment or be unexported

Reply:
this is a backport and we should not add something that is not in master